code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +92,50 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	var/ref_client = "\ref[src]"
+	for(var/datum/adminticket/T in admintickets)

```

Comment:
Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)